templates/home/deprecated/home-v2-fr.html

```diff
@@ -0,0 +1,332 @@
+	<title>Accueil - Canada.ca - Canada.ca</title>
+	<meta content="width=device-width, initial-scale=1" name="viewport">
+	<meta name="description" content=" " />
+	<link href="/dist/GCWeb/assets/favicon.ico" rel="icon" type="image/x-icon" />
+	<link rel="stylesheet" href="/dist/GCWeb/css/theme.css" />
+	<noscript>
+		<link rel="stylesheet" href="/dist/wet-boew/css/noscript.css" />
+	</noscript>
+</head>
+
+<body class="home page-type-nav" vocab="http://schema.org/" typeof="WebPage">
+	<nav>
+		<ul id="wb-tphp">
+			<li class="wb-slc"><a class="wb-sl" href="#wb-cont">Passer au contenu principal</a></li>
+			<li class="wb-slc visible-sm visible-md visible-lg"><a class="wb-sl" href="#wb-info">Passer à «&#160;À propos de ce site&#160;»</a></li>
+		</ul>
+	</nav>
+	<header>
+		<div id="wb-bnr" class="container">
+			<div class="row">
+				<section id="wb-lng" class="col-xs-3 col-sm-12 pull-right text-right">
+					<h2 class="wb-inv">Sélection de la langue</h2>
+					<ul class="list-inline mrgn-bttm-0">
+						<li>
+							<a lang="en" hreflang="en" href="contextual-signin-home-en.html">
+								<span class="hidden-xs">English</span>
+								<abbr title="English" class="visible-xs h3 mrgn-tp-sm mrgn-bttm-0 text-uppercase">en</abbr>
+							</a>
+						</li>
+					</ul>
+				</section>
+				<div class="brand col-xs-9 col-sm-5 col-md-4" property="publisher" typeof="GovernmentOrganization">
+					<img src="/dist/GCWeb/assets/sig-blk-fr.svg" alt="Gouvernement du Canada" property="logo" /><span class="wb-inv"> / <span lang="en">Government of Canada</span></span>
+					<link property="url" href="https://wet-boew.github.io/GCWeb/" />
+					<meta property="name" content="Gouvernement du Canada">
+					<meta property="areaServed" typeof="Country" content="Canada" />
+					<link property="logo" href="/dist/GCWeb/assets/wmms-blk.svg" />
+				</div>
+				<section id="wb-srch" class="col-lg-offset-4 col-md-offset-4 col-sm-offset-2 col-xs-12 col-sm-5 col-md-4">
+					<h2>Recherche</h2>
+					<form action="#" method="post" name="cse-search-box" role="search">
+						<div class="form-group wb-srch-qry">
+							<label for="wb-srch-q" class="wb-inv">Rechercher dans Canada.ca</label>
+							<input id="wb-srch-q" list="wb-srch-q-ac" class="wb-srch-q form-control" name="q" type="search" value="" size="34" maxlength="170" placeholder="Rechercher dans Canada.ca" />
+							<datalist id="wb-srch-q-ac"></datalist>
+						</div>
+						<div class="form-group submit">
+							<button type="submit" id="wb-srch-sub" class="btn btn-primary btn-small" name="wb-srch-sub">
+								<span class="glyphicon-search glyphicon"></span>
+								<span class="wb-inv">Recherche</span>
+							</button>
+						</div>
+					</form>
+				</section>
+			</div>
+		</div>
+		<hr>
+		<div class="container">
+			<div class="row">
+				<div class="col-sm-9">
```

Comment:
This is the CSS grid configuration for the version 3 not for the version 2 which should say `<div class="col-sm-12">`